cluster/mesos/docker/docker-compose.yml

```diff
@@ -149,9 +156,29 @@ scheduler:
     --cluster-dns=10.10.10.10
     --cluster-domain=cluster.local
     --v=2
+  environment:
+  - MESOS_DOCKER_ETCD_TIMEOUT
+  - MESOS_DOCKER_MESOS_TIMEOUT
+  - MESOS_DOCKER_API_TIMEOUT
   links:
   - etcd
   - mesosmaster1
   - mesosslave1
   - mesosslave2
   - apiserver
+sslgen:
+  image: mesosphere/kubernetes-mesos-sslgen
+  entrypoint: [ "k8sm-sslgen.sh" ]
+  command: [ "/var/run/kubernetes/auth" ]
+  volumes:
+  - ${MESOS_DOCKER_VOLUME_ROOT}/auth:/var/run/kubernetes/auth
```

Comment:
kubectl configuration still needs access to the `root-ca.crt` and `token-users`, but now that I think about it, we can probably use `docker cp` to get them out.